drivers/vertica/vertica.go

```diff
@@ -30,6 +79,35 @@ func init() {
 			}
 			return ver, nil
 		},
+		Open: func(_ context.Context, u *dburl.URL, stdout, stderr func() io.Writer) (func(string, string) (*sql.DB, error), error) {
+			return func(_ string, params string) (*sql.DB, error) {
+				queries := u.Query()
+
+				if queries.Get("tlsmode") == "server-strict" && hasCustomTlsConfig(queries) {
```

Comment:
Avoid silently ignoring parameters and return an actionable error message instead.
```suggestion
				if hasCustomTlsConfig(queries) {
				    if queries.Get("tlsmode") != "server-strict" {
				        return nil, fmt.Errorf("error: when ca_path is set, tlsmode must be set to server-strict")
				    }
```